app/vulncode_app/models.py

```diff
@@ -73,6 +76,7 @@ class Package(models.Model):
     A software package with minimal identifying information.
     Other identifiers are stored as PackageReference.
     """
+    impacted_package = models.ManyToManyField('ImpactedPackage')
```

Comment:
Is this really a ManyToMany here? Can an ImpactedPackage be resolved by many packages?
if so the name should be plural `impacted_packages`... since this will resolve to many not one. Hence you likely name collision source?